beancards/src/main/java/com/bbdgrads/beancards/Entities/Player.java

```diff
@@ -8,43 +8,43 @@
 import jakarta.persistence.Table;
 
 @Entity
-@Table(name = "players")
+@Table(name = "Players")
 public class Player {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    @Column(name = "id")
-    private Integer id;
-    @Column(name = "username")
-    private String userName;
-    @Column(name = "logedin")
-    private boolean logedIn;
-
+    @Column(name = "PlayerId")
+    private Integer playerId;
+    @Column(name = "DisplayName")
+    private String displayName;
     protected Player() {}
 
-    public Player(String userName) {
-        this.userName = userName;
+    public Player(String displayName) {
+        this.displayName = displayName;
     }
 
     @Override
     public String toString() {
         return String.format(
-            "Player[id=%d, userName='%s']",
-            id, userName);
+            "Player[playerId=%d, displayName='%s']",
+            playerId, displayName);
     }
 
     public Integer getId() {
-        return id;
+        return playerId;
```

Comment:
I think leave it id as it is in player table right .. And in composite tables we could use playerId like in Trade entity